code/game/objects/items/food/deepfried.dm

```diff
@@ -36,6 +36,11 @@
 	if(!(SEND_SIGNAL(fried, COMSIG_ITEM_FRIED, src) & COMSIG_FRYING_HANDLED)) //If frying is handled by signal don't do the defaault behavior.
```

Comment:
Wait, shouldn't we use the signal instead?

Reply:
![fdf3b9e737eb2dca09fbfbb2](https://user-images.githubusercontent.com/40974010/204185208-0cef59a8-5c97-4afa-9129-da035f22af3b.jpg)